icu4c/source/i18n/decimfmt.cpp

```diff
@@ -821,113 +1092,184 @@ void DecimalFormat::setScientificNotation(UBool useScientific) {
 }
 
 int8_t DecimalFormat::getMinimumExponentDigits(void) const {
+    // Not much we can do to report an error.
+    if (fields == nullptr || fields->properties.isNull()) {
+        // Fallback to using the default instance of DecimalFormatProperties.
+        return DecimalFormatProperties::getDefault().minimumExponentDigits;
```

Comment:
You don't need a static_cast here like on the following line?  If you don't, consider deleting the static_cast below.